runtime/gc_glue_java/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -721,17 +721,17 @@ MM_CollectorLanguageInterfaceImpl::scavenger_switchConcurrentForThread(MM_Enviro
 			j9gs_disable(&vmThread->gsParameters);
 		}
 		/*
-		 * By setting evacuateTop to NULL and evacuateBase to ~evacuateTop
+		 * By setting readBarrierRangeCheckTop to NULL and readBarrierRangeCheckBase to ~readBarrierRangeCheckTop
```

Comment:
This comment isn't accurate with the code below.  Can you update it?

Reply:
updated